src/main/java/com/keeper/homepage/domain/member/application/MemberService.java

```diff
@@ -60,4 +64,11 @@ public void unfollow(Member member, long otherId) {
     Member other = memberFindService.findById(otherId);
     member.unfollow(other);
   }
+
+  public MemberProfileResponse getMemberProfile(long memberId) {
+    Member findMember = memberFindService.findById(memberId);
+    List<Friend> findAllFollowers = friendRepository.findAllByFollowee(findMember);
+
+    return MemberProfileResponse.of(findMember, findAllFollowers);
```

Comment:
```suggestion
    Set<Friend> follower = findMember.getFollower();
    Set<Friend> followee = findMember.getFollowee();

    return MemberProfileResponse.of(findMember, follower, followee);
```

이렇게 팔로워와 팔로잉을 조회하고, `of` 메서드로 넘긴 뒤